bundles/basic_modes/mode_definitions.moon

```diff
@@ -293,7 +293,8 @@ common_auto_pairs = {
     }
 
   python:
-    extensions: { 'sc', 'py', 'pyw' }
+    extensions: { 'sc', 'py', 'pyw', 'pyx' }
+    patterns: { 'wscript$', 'SConstruct', 'SConscript', 'BUILD' }
```

Comment:
@nilnor 

1. I forgot the anchors...
2. I'll remove `BUILD`, though I don't know of any build systems that use it except for Bazel (uses a subset of Python 2) and Pants (uses Python, 3 I believe).